Firestore/Source/Public/FIRFieldValue.h

```diff
@@ -38,6 +38,27 @@ NS_SWIFT_NAME(FieldValue)
  */
 + (instancetype)fieldValueForServerTimestamp NS_SWIFT_NAME(serverTimestamp());
 
+/**
+ * Returns a sentinel that can be used with setData() or updateData() to union one or more
```

Comment:
"sentinel" is a kind of implementation detail word that I don't think actually matters to our end users.

Maybe start with something like:

Returns a special value that can be used with setData() or updateData() that tells the server to union the given elements with any array value that already exists on the server.

Reply:
SGTM